packages/core/src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -95,14 +95,15 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
                 await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
             }
 
+            const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}`, { params: { method: methodId } }));
```

Comment:
I think we're already loading the payment method in line #107. Wouldn't we removing one?

Reply:
we do for non vaulted, by moving it to this line, we get it initialized for vaulted too, so we can have the quote available for the level3 data

Reply:
can't we just pass `gatewayId ` as is?

Reply:
@FranciscoOnt  passing it directly would not work because its an optional